release/tests/test_get_token.py

```diff
@@ -0,0 +1,39 @@
+"""
+Tests for testeng-ci/release/get-token
+"""
+from mock import patch, mock_open
+from unittest import TestCase
+
+# Ensure we're properly handling python2 vs 3
+from sys import version_info
+if version_info.major == 2:
+    import __builtin__ as builtins  # pylint: disable=import-error
+else:
+    import builtins  # pylint: disable=import-error
+
+from release.get_token import get_token
+
+
+class GetTokenTestCase(TestCase):
```

Comment:
Should also have tests for when the token is available in multiple ways (test that the one we want is used), and a test for when no token is found.